transformer_engine/pytorch/tensor/mxfp8_tensor.py

```diff
@@ -384,6 +384,7 @@ def _set_data(self, tensor: torch.Tensor) -> None:
 
         # Quantize to FP8
         assert self._quantizer is not None, "Can't quantize without a quantizer"
+        self._quantizer.internal = False
```

Comment:
This makes me think that internal should maybe be an option to tex.quantize rather than the member of quantizer.